### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by reneemes**
This PR updates the:

* CSS stylesheet to include button and curser interactions, and flexbox on the items view.
* Updated the merchant sections in the main.js file to include the words "edit" and "delete" next to the icons on the buttons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="style.css">
```
Comment (jchirch) [inline index.html:6]: Links style sheet to the html. Nice job figuring it out.

File: style.css [hunk 2]
```diff
@@ -5,26 +5,36 @@
...
-  background-color: gray;
+  background-color: steelblue;
 }
 
 body {
   display: flex;
   margin: 0px;
 }
 
+button {
```
Comment (jchirch) [inline style.css:16]: Looks perfect in application, code is surprisingly simple

File: style.css [hunk 3]
```diff
@@ -83,6 +144,16 @@ form {
...
+#items-view {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.item h2, p {
+  text-align: center;
+}
+
```
Comment (stefanjshepard) [inline style.css:156]: Flexbox here meets requirement.

File: style.css [hunk 4]
```diff
@@ -83,6 +144,16 @@ form {
   justify-content: center;
 }
 
+#items-view {
```
Comment (jchirch) [inline style.css:147]: flex wrap had a huge impact on the UI experiance

**Review by jchirch — COMMENTED** [review]
fantastic job!

**Pull request merged**